releases/release-5.4.1.md

```diff
@@ -0,0 +1,234 @@
+    - (dup: release-5.2.4.md > Bug fixes> TiFlash)- Fix a bug that MPP tasks might leak threads forever [#4238](https://github.com/pingcap/tiflash/issues/4238)
+    - (dup: release-5.2.4.md > Bug fixes> TiFlash)- Fix the overflow that occurs when casting `FLOAT` to `DECIMAL` [#3998](https://github.com/pingcap/tiflash/issues/3998)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiFlash)- Fix the issue that expired data is recycled slowly [#4146](https://github.com/pingcap/tiflash/issues/4146)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiFlash)- Fix a bug that canceled MPP queries might cause tasks to hang forever when the local tunnel is enabled [#4229](https://github.com/pingcap/tiflash/issues/4229)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiFlash)- Fix the issue of memory leak that occurs when a query is canceled [#4098](https://github.com/pingcap/tiflash/issues/4098)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiFlash)- Fix the issue of memory leak that occurs when a query is canceled [#4098](https://github.com/pingcap/tiflash/issues/4098)
+    - (dup: release-5.2.4.md > Bug fixes> TiFlash)- Fix the wrong result that occurs when casting `DATETIME` to `DECIMAL` [#4151](https://github.com/pingcap/tiflash/issues/4151)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiFlash)- Fix the potential issue of TiFlash panic when `Snapshot` is applied simultaneously with multiple DDL operations [#4072](https://github.com/pingcap/tiflash/issues/4072)
+    - (dup: release-5.2.4.md > Bug fixes> TiFlash)- Fix the bug that invalid storage directory configurations lead to unexpected behaviors [#4093](https://github.com/pingcap/tiflash/issues/4093)
+    - (dup: release-5.2.4.md > Bug fixes> TiFlash)- Fix the bug that some exceptions are not handled properly [#4101](https://github.com/pingcap/tiflash/issues/4101)
+    - (dup: release-5.2.4.md > Bug fixes> TiFlash)- Fix the issue that casting `INT` to `DECIMAL` might cause overflow [#3920](https://github.com/pingcap/tiflash/issues/3920)
+    - (dup: release-5.2.4.md > Bug fixes> TiFlash)- Fix the issue that the result of `IN` is incorrect in multi-value expressions [#4016](https://github.com/pingcap/tiflash/issues/4016)
+    - (dup: release-5.2.4.md > Bug fixes> TiFlash)- Fix the issue that the date format identifies `'\n'` as an invalid separator [#4036](https://github.com/pingcap/tiflash/issues/4036)
+    - (dup: release-5.2.4.md > Bug fixes> TiFlash)- Fix the potential query error after adding columns under heavy read workload [#3967](https://github.com/pingcap/tiflash/issues/3967)
+    - (dup: release-5.2.4.md > Bug fixes> TiFlash)- Fix the panic issue that occurs when the memory limit is enabled [#3902](https://github.com/pingcap/tiflash/issues/3902)
+
++ Tools
+
+    + Backup & Restore (BR)
+        - (dup: release-5.2.4.md > Bug fixes> Tools> Backup & Restore (BR))- Fix the issue that BR fails to back up RawKV [#32607](https://github.com/pingcap/tidb/issues/32607)
+
+    + TiCDC
+
```

Comment:
- Fix the DML construct error issue caused by `rename tables` DDL. [#5059](https://github.com/pingcap/tiflow/issues/5059)
- Fix a rare likelihood that replication be stuck if the owner is changed when the new scheduler is enabled (disabled by default). [#4963](https://github.com/pingcap/tiflow/issues/4963)
- Fix ErrProcessorDuplicateOperations when new scheduler is enabled (disabled by default) [#4769](https://github.com/pingcap/tiflow/issues/4769)
- Fix the issue that ticdc failed to start when connects to multiple pd endpoints with tls-enabled and the 1st endpoint is not available [#4777](https://github.com/pingcap/tiflow/issues/4777)
- Fix checkpoint metrics when tables are being scheduled. [#4714](https://github.com/pingcap/tiflow/issues/4714)